components/git/epilogue.js

```diff
@@ -11,6 +11,7 @@ $ ncu-config set branch master   # Assuming you are landing commits on master
 $ git checkout master
 $ git node land --abort          # Abort a landing session, just in case
 $ git node land $PRID            # Start a new landing session
+$ git node land $URL             # Start a new landing session
```

Comment:
Can you put something down in the comment indicating this is an *alternative* way to land a PR, not a necessary step? Something like

```
git node land $URL             # Or, start a new landing session using the PR URL
```